automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -89,4 +107,16 @@ object Leonardo extends WorkbenchClient with LazyLogging {
     }
 
   }
+
+  object notebooks {
+    def notebooksPath(googleProject: GoogleProject, clusterName: ClusterName): String =
+      s"notebooks/${googleProject.string}/${clusterName.string}"
+
+    def get(googleProject: GoogleProject, clusterName: ClusterName)(implicit token: AuthToken, webDriver: WebDriver): NotebooksListPage = {
```

Comment:
I'm not sure this belongs here. It's just constructing a page object, not making a web service call.

Reply:
Yeah, this is not a simple GET like you would expect.

However `GET /notebooks/foo/bar` in the Leo API doesn't return data like a normal GET, either.  Instead it opens a proxy WebSocket connection to the notebook server.

This was my attempt to represent that behavior here - but it may be more appropriate to drive the actual proxy here.